service/controller/resource/monitoring/remotewriteingress/v1beta1/client.go

```diff
@@ -0,0 +1,29 @@
+package v1beta1
```

Comment:
you can also use one generic client. and eventually you check the APIVersion

Reply:
I really do not think so but you can try it yourself and we see how it goes :)

Reply:
It its possible sure :)

Reply:
I do not think focusing on v1beta1 makes much sense here